### PR TITLE
Add mini36

### PR DESCRIPTION
**Pull request by controller-works**
Add support for mini36 keyboard by Controller Works.

## Description

The mini36 is a low profile, split, ergonomic keyboard based on the RP2040 processor. It has a direct scan design, per-key LEDs and LED underglow, OLEDs on each half, and over current and ESD protection on the USB and TRRS connectors. Includes default and via keymaps.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/controllerworks/mini36/config.h [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#define SPLIT_LAYER_STATE_ENABLE
+#define SPLIT_LED_STATE_ENABLE
+#define SPLIT_MODS_ENABLE
+
+#undef I2C_DRIVER
+#define I2C_DRIVER I2CD1
+#undef I2C1_SCL_PIN
+#define I2C1_SCL_PIN GP25
+#undef I2C1_SDA_PIN
+#define I2C1_SDA_PIN GP24
```
Comment (zvecr) [inline keyboards/controllerworks/mini36/config.h:41]:
```suggestion
#define I2C_DRIVER I2CD1
#define I2C1_SCL_PIN GP25
#define I2C1_SDA_PIN GP24
```

File: keyboards/controllerworks/mini36/mini36.h [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/*
```
Comment (zvecr) [inline keyboards/controllerworks/mini36/mini36.h:1]: This file can be removed

File: keyboards/controllerworks/mini36/readme.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+| KEY13  | GP18 | 2         | 3           | 6         | 3            |
+| KEY14  | GP19 | 2         | 4           | 6         | 4            |
+| KEY15  | GP20 | 3         | 0           | 7         | 0            |
+| KEY16  | GP21 | 3         | 1           | 7         | 1            |
+| KEY17  | GP22 | 3         | 2           | 7         | 2            |
+| SDA    | GP24 |           |             |           |              |
+| SCL    | GP25 |           |             |           |              |
+
+
+\* Right Column numbers run right (pinky position) to left (thumb position)
```
Comment (zvecr) [inline keyboards/controllerworks/mini36/readme.md:73]:
```suggestion
```

File: keyboards/controllerworks/mini36/info.json [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+            ["GP15", "GP16", "GP17", "GP18", "GP19"],
+            ["GP20", "GP21", "GP22",   null,   null]
+        ]
+    },
+    "rgblight": {
+        "pin": "GP0"
+     },
+    "split": {
+        "enabled": true,
+        "main": "left",
```
Comment (zvecr) [inline keyboards/controllerworks/mini36/info.json:31]: Currently doesnt map to anything
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/controllerworks/mini36/mini36.h (hunk 2 above)
Comment (controller-works) [reply keyboards/controllerworks/mini36/mini36.h:1]: Removed.

**Comment by controller-works** [comment]
@zvecr I believe I implemented all of the suggested changes. Thank you for your time in reviewing this PR.

File: keyboards/controllerworks/mini36/readme.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# mini36
+
+![mini36](https://i.imgur.com/JwhiE9l.png)
+![mini36PCB](https://i.imgur.com/6NjZ8Cq.jpg)
```
Comment (drashna) [inline keyboards/controllerworks/mini36/readme.md:4]:
```suggestion
![mini36](https://i.imgur.com/JwhiE9ll.png)
![mini36PCB](https://i.imgur.com/6NjZ8Cql.jpg)
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by controller-works** [comment]
@drashna All changes are implemented as requested. Thank you for your time in reviewing this PR.

**Review by drashna — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**